connectors/sources/generic_database.py

```diff
@@ -152,6 +152,9 @@ async def execute_query(self, query_name, fetch_many=False, **query_kwargs):
         Yields:
             list: Column names and query response
         """
+        if self.queries is None:
+            raise NotImplementedError()
```

Comment:
It makes sense to raise error if `self.queries` is None, but `NotImplementedError`?

Reply:
`self.queries` is set to None in this base class. If the concrete class does not set it and this function is called, it won't work. So in that case it means the concrete class is incomplete. And for incomplete concrete class, you use `NotImplementedError`